lib/dispatch/dispatch.ex

```diff
@@ -32,7 +32,7 @@ defmodule Dispatch do
   @doc """
   Returns a list of usernames that should be request to review the pull request
   """
-  def fetch_selected_users(repo, stacks, author_username) do
+  def fetch_selected_users(repo, stacks, author_username, mute_learners \\ false) do
```

Comment:
We don’t need a default value, since we always pass the argument.

```suggestion
  def fetch_selected_users(repo, stacks, author_username, mute_learners) do
```

Reply:
`fetch_selected_users` is used in a lot of unit tests and I don’t think we should explicitly pass a flag that is meant to disable a functionality.